### PR TITLE
Removing abbreviations in 10.13.0

### PR DESCRIPTION
**Pull request by MWindel**
- Cherry-picked the merge commit e77c2f0 
- Removed remaining cases of "e.g.", "i.e." and "etc." manually
- See https://cumulocity.atlassian.net/browse/MTM-43516 for more information

### CODE REVIEW

File: content/machine-learning/casting-defect-detection-bundle/overview.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ aliases:
...
+A casting defect is an undesired irregularity in a metal casting process. There are many types of defects in casting such as blow holes, pinholes, burr, shrinkage defects, mould material defects, pouring metal defects, metallurgical defects, and so on. Defects are an unwanted thing in the casting industry. For removing this defective product, all industry have their quality inspection department. But the main problem is that this inspection process is carried out manually. It is a very time-consuming process and due to human accuracy, this is not 100% accurate. This can lead to the rejection of the whole order which creates a big loss to the company.
```
Comment (BeateRixen) [inline content/machine-learning/casting-defect-detection-bundle/overview.md:12]: I think we can leave out "and so on" as we already have "such as"

File: content/machine-learning/web-app-mlw-bundle/third-party-data-pull.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ layout: redirect
 weight: 35
 ---
 
-Machine Learning Workbench (MLW) provides connectors to the various data sources such as {{< product-c8y-iot >}}, DataHub, etc. from where the data could be downloaded to start the machine learning model-building process.
+Machine Learning Workbench (MLW) provides connectors to the various data sources such as {{< product-c8y-iot >}}, DataHub, and so on from where the data could be downloaded to start the machine learning model-building process.
```
Comment (BeateRixen) [inline content/machine-learning/web-app-mlw-bundle/third-party-data-pull.md:7]: Again, we have "such as", so "and so on" not really required

**Review by rssag — APPROVED** [review]

File: content/machine-learning/casting-defect-detection-bundle/overview.md (hunk 1 above)
Comment (MWindel) [reply content/machine-learning/casting-defect-detection-bundle/overview.md:12]: Done

File: content/machine-learning/web-app-mlw-bundle/third-party-data-pull.md (hunk 2 above)
Comment (MWindel) [reply content/machine-learning/web-app-mlw-bundle/third-party-data-pull.md:7]: Done

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**